jenkins/flow/master/edx-platform-bok-choy-master.groovy

```diff
@@ -35,39 +36,39 @@ try{
   guard{
     parallel(
         {
-          bok_choy_1 = build(subsetJob, sha1: sha1, SHARD: "1", TEST_SUITE: "bok-choy", ENV_VARS: params["ENV_VARS"], PARENT_BUILD: "master #" + build.number)
+          bok_choy_1 = build(subsetJob, sha1: sha1, SHARD: "1", TEST_SUITE: "bok-choy", ENV_VARS: params["ENV_VARS"], PARENT_BUILD: "master #" + build.number, WORKER_LABEL: workerLabel)
           toolbox.slurpArtifacts(bok_choy_1)
```

Comment:
@cpennington We looked at that before and there was some reason why it wouldn't work; perhaps it was a problem with the build-flow DSL. That said, if I can't remember, it might be worth a cycle looking at it again in another PR.